src/stylesheets/components/_list-styles.scss

```diff
@@ -168,6 +168,28 @@ dl, dt, dd {
   }
 }
 
+// Numbered / Roman Numeral / Alphabetical Lists
+
+.list--number, .list--roman, .list--alpha {
+  @extend .list;
+  list-style-position: inside;
```

Comment:
set to `outside` else bug WEBDEV-354 will be re-introduced

Reply:
List position removed and left margin applied that will handle up to 999 items nicely